Purchases/Purchasing/StoreKitAbstractions/StoreProduct.swift

```diff
@@ -186,6 +190,52 @@ public extension StoreProduct {
 
 }
 
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
+public extension StoreProduct {
+    /// Asynchronously determines the subset of ``discounts`` that's eligible for the current user.
```

Comment:
kinda feel like `Asynchronously` here is redundant - the API already says `async`, and most of our methods are async anway

Reply:
That's fair. I wanted to make it clear that this made API requests, but I guess you're right.

Reply:
was thinking maybe we should rename this to `getEligibleDiscounts` to make more clear that it is async, but then i realized we don't really follow that convention for async functions, only for those with completion blocks? is that by design?

Reply:
Actually I think that's a great way to convey that this is a potentially "expensive" operation instead of a simpler property getter 👍

Reply:
Changed comment and method name.